python/lsst/ts/Dome/dome_csc.py

```diff
@@ -201,27 +219,23 @@ async def write_then_read_reply(self, command, **params):
         Returns
         -------
         configuration_parameters : `dict`
-            A dict of the form {"reply": {"param1": value1, "param2": value2}} where "reply" can for
-            instance be "OK" or "ERROR".
+            A dict of the form {"reply": {"param1": value1, "param2": value2}}
+            where "reply" can for instance be "OK" or "ERROR".
```

Comment:
Is this still correct? I thought EIE had switched to having a key for the reply (e.g. `{"reply": reply, ...}`. Or is that only for commands, not replies?

Reply:
Good catch! This proves that it is much better to have the documentation in .rts files rather than on Confluence. I have updated the doc string.